Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/ui/main/MainFragment.kt

```diff
@@ -67,6 +73,19 @@ class MainFragment : Fragment() {
         showOneTimeTracingExplanationDialog()
     }
 
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+
+        if (errorResetTool.isResetNoticeToBeShown) {
```

Comment:
How about moving this block to a dedicated function "showResetNoticeIfNeeded"

Reply:
> How about moving this block to a dedicated function "showResetNoticeIfNeeded"

I don't like methods that are only called from one location. Use doc to make more readable.